src/index.d.ts

```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
          */
         stubNull():  null;
 
+        /**
+         * A function that returns new empty string.
+         */
+        stubString(): '';
```

Comment:
Again, infering type not the value ?

Reply:
Is this a problem, typescript handles this fine?

Reply:
If it handles it, it's fine. Did not know about literal types in typescript.